index.bs

```diff
@@ -2512,8 +2512,8 @@ is called on.
 
 	1. Let <var>o</var> be a new object of type <var>n</var>.
 
-	2. Let <var>option</var> be a dictionary of the type <a href="#dfn-associated-option-object">associated</a> to the interface
-		<a href="#dfn-associated-interface">associated</a> to this factory method.
+	2. Let <var>option</var> be a dictionary of the type <a href="#associated-option-object">associated</a> to the interface
```

Comment:
These should be turned into autolinks instead.

Reply:
Aargh. I keep getting `{{` and `[[` confused.  Using `[[` works great.

Done.